Userland/Libraries/LibC/ulimit.cpp

```diff
@@ -1,27 +1,28 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2022, Lucas Chollet <lucas.chollet@free.fr>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #include <AK/Format.h>
 #include <assert.h>
 #include <sys/resource.h>
+#include <syscall.h>
 #include <ulimit.h>
 
 extern "C" {
 
 long ulimit([[maybe_unused]] int cmd, [[maybe_unused]] long newlimit)
 {
-    dbgln("FIXME: Implement getrusage()");
```

Comment:
Why is the dbgln being removed if the function still crashes? Please update the message instead.